### PR TITLE
Display version on results

### PR DESCRIPTION
**Pull request by waghanza**
Hi,

This `PR` add version, only `X.Y`, for each frameworks implementations.

The language version will be added later, the language version depends on :
+ Which provider is use (`aws`, `gcp`, `azure`, `digitalocean` ...)
+ Operating system (`debian`, `centos` ...)
+ ...

Regards,

### CODE REVIEW

**Comment by waghanza** [comment]
@tbrand and all `gophers` **here** How can I display version in `golang` ?

Using `import` directly download from **master**. How can I represent version number ?

**Comment by waghanza** [comment]
@tbrand @OvermindDL1 @aichholzer I've added a sample resultset here => https://github.com/waghanza/http-benchmark/tree/version_display

What do you think  ?

PS : Display version of language will be added after cloudification, cause language version (at least for some languages such as `ruby` depends on `cloud` **provider** / **os**)

File: go/gin/Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ FROM golang
 WORKDIR /usr/src/app
 COPY main.go .
 
-RUN go get -u github.com/gin-gonic/gin
+RUN go get -u github.com/gin-gonic/gin@1.3
```
[the remark this answers is not in the document]
Comment (waghanza) [reply go/gin/Dockerfile]: probably, as I never done (**production-ready**) go app, I do not understand what you want to do, but feel free to open a separated `PR` / `issue`, we will discuss about that

[the remark this answers is not in the document]
Comment (appleboy) [reply go/gin/Dockerfile]: See https://github.com/the-benchmarker/web-frameworks/pull/377

**Comment by waghanza** [comment]
@OvermindDL1 I have re-made result (https://github.com/waghanza/http-benchmark/tree/version_display#results)
How can I fix version for `evhtp` ?

**Comment by OvermindDL1** [comment]
Well the version it is using is `1.2.16-p4` and that won't change until the huntergate is updated in it's dependency block, so that won't be changing.  You can always query the version from the build file (to write out to a file or whatever) via `HUNTER_libevhtp_VERSION` version of course.

File: README.md [hunk 2]
```diff
@@ -82,64 +82,68 @@ CPU Cores: 4
 :two: rocket (rust)
 
 
-:three: rack-routing (ruby)
+:three: laravel (php)
```
Comment (appleboy) [inline README.md]: I can't magic that the laravel framework wins a third of the list.

**Comment by waghanza** [comment]
@appleboy sure ... The results are not representative yet (cause of docker).
The only thing thanks will not change is the way how infos are shown. 

Probably for Xmas / end of year, results will be produced of digitalocean (si Docker fluctuation will be thrown away)

**Pull request merged**